portfolio/src/main/webapp/script.js

```diff
@@ -12,16 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
+// Adds "Hello Andrew" to page 
+async function getGreetingAsyncAwait() {
```

Comment:
you don't have to mention "asyncAwait" in the name of the method.

Reply:
No problem!